src/Containers.jsx

```diff
@@ -707,28 +707,31 @@ class Containers extends React.Component {
 
         const createContainer = (inPod) => {
             if (localImages)
-                Dialogs.show(<ImageRunModal user={this.props.user}
-                                            localImages={localImages}
-                                            pod={inPod}
-                                            registries={this.props.registries}
-                                            selinuxAvailable={this.props.selinuxAvailable}
-                                            podmanRestartAvailable={this.props.podmanRestartAvailable}
-                                            userPodmanRestartAvailable={this.props.userPodmanRestartAvailable}
-                                            systemServiceAvailable={this.props.systemServiceAvailable}
-                                            userServiceAvailable={this.props.userServiceAvailable}
-                                            userLingeringEnabled={this.props.userLingeringEnabled}
-                                            onAddNotification={this.props.onAddNotification}
-                                            version={this.props.version} />);
+                Dialogs.show(
+                    <utils.PodmanInfoContext.Consumer>
+                        {(podmanInfo) => (
```

Comment:
Hmm, that's also annoying boilerplate that has to be dragged around. Is it possible to pull that up to the top-level component somehow? But hmm, I figure you somehow need to get a handle to `podmanInfo`..

Reply:
You can if you use functional components, but for combining multiple context providers for normal classes you have to use these shenanigans.